### DISCUSSION TITLE
GitHub 登录后无法显示头像，在文档中评论会被自动删除

### DISCUSSION THREAD

**Discussion by zywvvd** [walinejs/waline#522] [Q&A] [answered]
GitHub 登录后无法显示头像
![](https://photos.zywvvd.com/win11-mt/20210903143246.png)
在文档中评论会被自动删除
文档链接：https://waline.js.org/guide/client/avatar.html#%E6%B3%A8%E6%84%8F%E4%BA%8B%E9%A1%B9
![](https://photos.zywvvd.com/win11-mt/20210903143333.png)
然后再刷新就没了
![](https://photos.zywvvd.com/win11-mt/20210903143401.png)

**Comment by lizheming** [comment, marked as answer]
官网之前被垃圾评论了，所以制定了一些奇怪的审核规则，你的评论命中了审核规则所以进到待审核列表里了，没有消失~

**Reply by zywvvd** [reply]
好吧，那怎么才能有头像啊啊啊啊~

**Reply by lizheming** [reply]
头像需要我之后看下，之前我已经发现这个问题了，可能是 Bug~~

**Reply by zywvvd** [reply]
好吧 加油~ 需要提供什么信息的话我这边可以帮忙
